### PR TITLE
chore(common): Update standards data

### PR DESCRIPTION
**Pull request by ermshiperete**
@keymanapp-test-bot skip

### CODE REVIEW

**Comment by mcdurdin** [comment]
We just have to be a little careful with testing in case metadata formats have changed.

**Comment by ermshiperete** [comment]
One thing that changed is in `iso639-3.tab` the `Language_Type` column now has `H` (Historical) instead of `A` (Ancient) for several languages. But I'm not sure that information is used anywhere in our code.

Other than that it's only updated data AFAICT.

File: resources/standards-data/readme.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ See also [ldml-keyboards/readme.md](ldml-keyboards/readme.md).
 Data from <https://www.unicode.org/Public/15.0.0/ucd/>:
```
Comment (darcywong00) [inline resources/standards-data/readme.md:61]: Is this also 
```suggestion
Data from <https://www.unicode.org/Public/15.1.0/ucd/>:
```

Comment (ermshiperete) [reply resources/standards-data/readme.md]: Done. Thanks!

**Comment by rc-swag** [comment]
Here are some observations using the Delphi debugger. However, I really don't understand the whole code.
There is an error about inserting dulplicates 
```
class procedure TISO6393ToBCP47Map.Fill(dict: TDictionary<string, string>);
begin
  dict.Add('aar','aa');
  dict.Add('abk','ab');
```
For the failing test of `sqt-YE` the following if statement in `CanonicalLanguageCodeUtils.FindBestTag` evaluates to true and exits. For the master branch this if statement evaluates to false. 

 ```
if not TLangTagsMap.LangTags.TryGetValue(t.Tag, LangTag) then
    begin
      // Not a known tag but perhaps it's a custom language
      // We'll make no further assumptions
      Exit(t.Tag);
    end;
```

**Comment by rc-swag** [comment]
The duplicate insertion area was the key here, due to duplicates in the langtag.json source, this meant the dictionary was not completely loaded resulting in these other tests failing. In other words those failed tests where a side effect of the duplicate data error.

**Comment by mcdurdin** [comment]
Failing tests due to data issues in langtags.json, reported at:
* https://github.com/silnrsi/langtags/issues/15
* https://github.com/silnrsi/langtags/issues/16

Note that most of the failing tests are caused by a cascade of errors rather than these root causes.

File: resources/standards-data/langtags/langtags.json [hunk 2]
```diff
@@ -93906,7 +94074,7 @@
         "full": "pht-Thai-TH",
         "iana": [ "Phu Thai" ],
         "iso639_3": "pht",
-        "latnnames": [ "Phu Thai or Phuu Thai", "Phu Thai or Phuu Thai" ],
+        "latnnames": [ "Phu Thai", "Phuu Thai" ],
```
Comment (mcdurdin) [inline resources/standards-data/langtags/langtags.json:94077]:
```suggestion
        "latnnames": [ "Phu Thai or Phuu Thai", "Phu Thai or Phuu Thai" ],
```

Per discussion in https://github.com/silnrsi/langtags/issues/17#issuecomment-2097803177, this is valid.

**Comment by ermshiperete** [comment]
Updated to the latest standards data which has the data fixed that previously required to handcraft `langtags.json`.

**Review by rc-swag — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.42-alpha](https://keyman.com/downloads/releases/alpha/18.0.42)

**Pull request merged**